libs/common/parameters_utils.h

```diff
@@ -164,6 +166,10 @@ static inline bool VtValueGetBool(const VtValue& value, bool defaultValue = fals
         VtArray<int> array = value.UncheckedGet<VtArray<int>>();
         return array.empty() ? false : (array[0] != 0);
     }
+    if (value.IsHolding<VtArray<unsigned int>>()) {
+        VtArray<unsigned int> array = value.UncheckedGet<VtArray<unsigned int>>();
```

Comment:
I think we can use the const reference to the object instead of a copy, even if I am guessing the underlying array buffer won't be copied anyway.